### PR TITLE
Adding main header

### PR DESCRIPTION
**Pull request by masterrom**
Custom Main Header is added to all the screens in the main Stack

### CODE REVIEW

File: src/screens/setup/create/ConfirmPassphrase.js [hunk 1]
```diff
@@ -172,51 +159,57 @@ class ConfirmPassphrase extends Component {
...
                 </View>    
 
-                </View>
+                <View style={styles.footerGrandparentContainer} >    
+                    <View style={styles.footerParentContainer} >
+                        <Text style={styles.textFooter} >Powered by ChainSafe </Text> 
+                    </View>  
+                </View> 
             </View>
             // <View style={styles.mainContainer}>
```
Comment (priom) [inline src/screens/setup/create/ConfirmPassphrase.js]: this block of code will not be used @CFDevelopment ?

File: src/screens/setup/create/ConfirmPassphrase.js [hunk 2]
```diff
@@ -266,22 +259,70 @@ const styles = StyleSheet.create({
...
+        height:20, 
+        width:20
     },
     tagContainer: {
         flexDirection: 'row',
         flexWrap: 'wrap',
-        padding: 2
+        alignItems: 'center',
+        marginLeft: '2.5%',
+
```
Comment (priom) [inline src/screens/setup/create/ConfirmPassphrase.js]: what's up with styling here @CFDevelopment ?

File: src/screens/setup/create/ConfirmPassphrase.js [hunk 3]
```diff
@@ -77,9 +77,9 @@ class ConfirmPassphrase extends Component {
             state.scrambledTags[x].selected = true;  
             // state.scrambledTags.push(tagItem);
```
Comment (priom) [inline src/screens/setup/create/ConfirmPassphrase.js]: what's happening here @masterrom ?

File: src/components/tokens/ListItem.js [hunk 4]
```diff
@@ -98,32 +98,30 @@ class CoinListItem extends Component {
...
             }
             containerStyle = {{
-              borderRadius: 10, 
+              shadowColor: 'grey',
+              shadowOffset: {width:0, height:0},
+              shadowRadius: 2,
+              shadowOpacity: 0.5,
               width: '100%', 
               height: 63,            
               backgroundColor: '#ffffff',
```
Comment (GregTheGreek) [inline src/components/tokens/ListItem.js]: remove inline styling!

File: src/screens/setup/create/ConfirmPassphrase.js [hunk 5]
```diff
@@ -172,88 +151,83 @@ class ConfirmPassphrase extends Component {
...
+                                    // return <Button
+                                    // style={styles.tag}
+                                    // title={item.word}
+                                    // key={item.index}
+                                    // onPress={() => this.addTag(item, "revert", index)} />
+                                })
+                            }
+                        </View> */}
+
+
```
Comment (GregTheGreek) [inline src/screens/setup/create/ConfirmPassphrase.js]: remove if we dont need it

**Pull request merged**